Firestore/Source/API/FIRFirestore.mm

```diff
@@ -243,6 +236,34 @@ - (void)ensureClientConfigured {
       FSTAssert(_settings.host, @"FirestoreSettings.host cannot be nil.");
       FSTAssert(_settings.dispatchQueue, @"FirestoreSettings.dispatchQueue cannot be nil.");
 
+      if (!_settings.enableTimestampsInSnapshots) {
```

Comment:
Can you add something like:

// TODO(var-const): DO NOT SUBMIT: Double-check these code snippets before submitting.

just to make sure we don't forget? :)  I don't want to be responsible for throwing a warning in every developer's faces with code that doesn't actually work. :-)

Reply:
Done.